vgc/cameraFilter.py

```diff
@@ -1,82 +1,107 @@
 """Filters a video stream.
```

Comment:
This should list all classes, exceptions, functions and objects that belong to the module. Add this. See "Dokumentering av kod" for more info.

Reply:
We have added this to the docstring.